src/commands/channels/configure.ts

```diff
@@ -0,0 +1,170 @@
+/*
+* Copyright 2022, Nuance, Inc. and its contributors.
+* All rights reserved.
+*
+* This source code is licensed under the Apache-2.0 license found in
+* the LICENSE file in the root directory of this source tree.
+*/
+
+import {flags} from '@oclif/command'
+import chalk from 'chalk'
+import makeDebug from 'debug'
+
+import * as ChannelsAPI from '../../mix/api/channels'
+import {ChannelBodyModality, ChannelModalities, ChannelModality, ChannelsConfigParams} from '../../mix/api/channels-types'
+import {MixClient, MixResponse, MixResult} from '../../mix/types'
+import MixCommand from '../../utils/base/mix-command'
+import {eInvalidValue} from '../../utils/errors'
+import * as MixFlags from '../../utils/flags'
+import {DomainOption, validateChannelColor, validateChannelModeOptions} from '../../utils/validations'
+
+const debug = makeDebug('mix:commands:channels:configure')
+
+export default class ChannelsConfigure extends MixCommand {
+  static description = `update channel details in a project
+
+Configure the modalities and color of an existing channel in a Mix project.
+If either value is not supplied, the existing property on the channel will be unmodified.
+
+A note on channel modes and colors:
+For your convenience, you may enter modes and colors case-insensitively.
+In particular, you can spell any given mode/color with lowercase or capital letters,
+and with dashes ('-') in place of underscores ('_'). The value will be internally converted
+into the format the Mix API expects before the request is made. As an example, the values
+'light-pink', 'light_Pink', and 'LIGHT-PINK' are all equivalent to Mix's 'LIGHT_PINK'.
+
+Acceptable channel modes are:
+audioscript
+dtmf
+interactivity
+richtext
+tts
+
+Acceptable channel colors are:
```

Comment:
The help for channels:create has a blank line before the list of modalities but not before the list of colors. In this command here, there is no blank link in either location. I don't have a strong opinion the blank line but it should be consistent within and across channels commands.